app/templates/components/forms/orders/order-form.hbs

```diff
@@ -279,12 +279,13 @@
         {{t 'Acknowledgement'}}
       </h4>
       <p>
-        {{t
-          'By clicking "Pay Now", I acknowledge that I have read and that I agree with all the terms of services and privacy policy of Open Event.'}}
+        {{t 'By clicking "{{orderState}}"' orderState=(if this.isPaidOrder (t "Proceed to Checkout") (t "Order Now"))}}
+        {{', I acknowledge that I have read and that I agree with the '}} 
```

Comment:
The entire message should be one string to make it translatable. 

`{{t 'By clicking "{{orderState}}", I acknowledge that I have read and that I agree with the {{terms}} and {{privacyPolicy}} on this website.' orderState=(if this.isPaidOrder (t "Proceed to Checkout") (t "Order Now') terms=... privacyPolicy=...}}`

Or else it will be separated in multiple different sentences which don't make sense in translation:

1. `By clicking "{{orderState}}"` ???
2. `, I acknowledge that I have read and that I agree with the ` umm?
3. ` and ` space surrounded and?
4. ` on this website.` what's the context?

When making things translatable by wrapping in `t` you have to make sure it makes sense to the translator